linode/manager
# fix: Make `yarn up:expose` actually expose Cloud Manager

## Description 📝
- Makes `yarn up:expose` actually expose Cloud Manager on all network interfaces
- This fix is useful for running Cloud Manager in Docker or other environments  

## How to test 🧪
- Run `yarn up:expose` from the root directory and verify Cloud Manager can be accessed from all interfaces
